src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -62,17 +72,7 @@ internal partial class SymbolTreeInfo : IChecksummedObject
 
         public bool ContainsExtensionMethod => _receiverTypeNameToExtensionMethodMap?.Count > 0;
 
-        /// <summary>
-        /// The task that produces the spell checker we use for fuzzy match queries.
-        /// We use a task so that we can generate the <see cref="SymbolTreeInfo"/> 
-        /// without having to wait for the spell checker construction to finish.
-        /// 
-        /// Features that don't need fuzzy matching don't want to incur the cost of 
-        /// the creation of this value.  And the only feature which does want fuzzy
-        /// matching (add-using) doesn't want to block waiting for the value to be
-        /// created.
-        /// </summary>
-        private readonly Task<SpellChecker> _spellCheckerTask;
+        private readonly SpellChecker _spellChecker;
```

Comment:
this used to be lazy when everything was in-proc and memory/cpu was very tight.  Now that this is all OOP, it's just much simpler to have no lazyness in the internal data of this index.  measuring things in the benchmarks showed no issue moving this to not be delayed.  Also, since this is used by add-using, it's highly likely for this to be needed almost immediately as the add-using codepath runs for tons of different user error scenarios.